crates/rustfix/tests/parse_and_replace.rs

```diff
@@ -6,65 +6,28 @@
 //! rustfix and applies the recommended suggestions to the `.rs` file. It then
 //! compares the result with the corresponding `.fixed.rs` file. If they don't
 //! match, then the test fails.
-//!
-//! There are several debugging environment variables for this test that you can set:
-//!
-//! - `RUST_LOG=parse_and_replace=debug`: Print debug information.
-//! - `RUSTFIX_TEST_BLESS=test-name.rs`: When given the name of a test, this
-//!   will overwrite the `.json` and `.fixed.rs` files with the expected
-//!   values. This can be used when adding a new test.
-//! - `RUSTFIX_TEST_RECORD_JSON=1`:  Records the JSON output to
-//!   `*.recorded.json` files. You can then move that to `.json` or whatever
-//!   you need.
-//! - `RUSTFIX_TEST_RECORD_FIXED_RUST=1`: Records the fixed result to
```

Comment:
We seem to remove this feature in this patch. I am fine living without it, but maybe we should call it out in the PR description?